python/lsst/pipe/tasks/insertFakes.py

```diff
@@ -555,6 +570,22 @@ def cleanCat(self, fakeCat):
         self.log.info("Removing %d rows with nBulge or nDisk outside of %0.2f <= n <= %0.2f" %
                       (badRows, minN, maxN))
 
+        if self.config.template:
+            try:
+                goodRows = (fakeCat["template"])
```

Comment:
`goodRows` and `badRows` is a bit unclear what you're actually doing here. I assume this is left over from another test but the rows for these settings aren't bad or good. They are just used or aren't.

Reply:
I've changed the names throughout this function as the point you make here is valid not just for this bit and I would like to keep the naming consistent.